src/main/java/com/mitchej123/hodgepodge/mixins/early/fml/MixinGuiModList.java

```diff
@@ -0,0 +1,62 @@
+package com.mitchej123.hodgepodge.mixins.early.fml;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import net.minecraft.client.gui.GuiScreen;
+
+import org.lwjgl.input.Mouse;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+import com.gtnewhorizon.gtnhlib.util.FilesUtil;
+
+import cpw.mods.fml.client.GuiModList;
+
+@Mixin(GuiModList.class)
+public abstract class MixinGuiModList extends GuiScreen {
+
+    int urlX = 0;
+    int urlY = 0;
+    String urlStr;
```

Comment:
These should all be private, prefixed with `hodgepodge$` and annotated with `@Unique`.